src/hocs/withTimeoutFallback/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DefaultFallback = () => (<span>Error loading</span>);
+
+const DEFAULT_OPTIONS = {
+  FallbackComponent: DefaultFallback,
+  timeout: 10000,
+};
+
+const TIMER_INTERVAL = 100;
+
+// TODO: CAKE-3226 Consider adding to react design system
```

Comment:
sounds like you've already considered it... ✅

Reply:
Removed! Thanks 😁